### DISCUSSION TITLE
Mod whitelists using CSteamID.

### DISCUSSION THREAD

**Discussion by Jdance-Media** [SmartlyDressedGames/Unturned-3.x-Community#3888] [Ideas]
GLSTs are associated with a SteamID. I think you know where I'm going with this.
![image](https://github.com/SmartlyDressedGames/Unturned-3.x-Community/assets/67397422/c86af44a-ce76-43b5-871e-7a64a2d07921)
Would be easier to just whitelist/blacklist all of the servers owned by an individual. This is mainly helpful with whitelists, as it would be difficult to obtain the GLST holder's SteamID of a server you are 'enemies' with in order to blacklist them.
Credit to Karma over on the Discord for mentioning that GLSTs are associated with a SteamID.

**Comment by SDGNelson** [comment]
Yeah that would probably make sense to do in the future. Back when the IP whitelist was added I'm not sure if Unturned even supported GSLT at the time, and then (last year?) started requiring it.
